gems/manageiq_foreman/manageiq_foreman.gemspec

```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |spec|
   spec.email         = ["keenan@thebrocks.net"]
   spec.description   = %q{Foreman Provider for ManageIQ}
   spec.summary       = %q{Foreman Provider for ManageIQ}
-  spec.homepage      = "http://github.com/manageiq/manageiq_foreman"
+  spec.homepage      = "https://github.com/ManageIQ/manageiq"
```

Comment:
For things like this, which are more tightly coupled to the application, I prefer the idea of keeping them in-repo... too frequently, their changes will need to be made in lock-step with a matching "use the new functionality" change in the app.

This semi-isolation is useful to keep track of our dependencies, and discourage "library" code from being too coupled... but however we colour it, this *is* part of the application codebase -- I don't think we can pretend otherwise.